ci/deploy-to-uat

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Echo all commands just before they are executed
+set -x
+
+# Stop processing this script if any of the commands we run return an error
+# (i.e. exit with a non-zero status)
+set -e
+
+cd /var/www
+
+# This script attempts a git fetch in CWP, waits for it to be ready
+# and creates a deployment request.
+
+# Creates a git fetch request
+curl -X POST -u $CWP_DASH_USER:$CWP_DASH_TOKEN \
+  https://dash.cwp.govt.nz/naut/project/$CWP_STACK_ID/git/fetches \
+  -H 'Content-Type: application/json' -H 'Accept: application/json' >response.json
+
+# We extract the request's status URL from the response
+LINK=$(cat response.json | jq -r '.data.links.self')
+
+# We extract the request's status URL from the response.
+# It will always be "Queued" at first.
+STATUS=$(cat response.json | jq -r '.data.attributes.status')
+
+# We use a while loop to wait a few seconds, and query CWP on the
+# fetch request. If the query has finished, our status will change
+# to Complete. If status is not achieved we loop again.
+while [ "$STATUS" != "Complete" ]; do
+  sleep 15
+  STATUS=$(curl --silent -X GET -u $CWP_DASH_USER:$CWP_DASH_TOKEN $LINK \
+    -H 'Content-Type: application/json' -H 'Accept: application/json' | jq -r '.data.attributes.status')
```

Comment:
```suggestion
  STATUS=$(
    curl --silent -X GET -u $CWP_DASH_USER:$CWP_DASH_TOKEN $LINK \
      -H 'Content-Type: application/json' -H 'Accept: application/json' \
      | jq -r '.data.attributes.status'
  )
```
🐘 I'd prefer we don't smoosh everything onto a couple of lines - I find it mores readable in this sort of layout.

Reply:
Odd. I think this was done by prettier or some other format. Either way, same situation as the comment above

Reply:
`prettier-plugin-sh` shouldn't have done that as it's using `shfmt` which doesn't format like this.

I'm not sure why this formatting change is blocked by your above comment, since we're only changing the formatting (which can be verified statically via `shellcheck`).

Reply:
It is not blocked but it will be rewritten properly as this piece of code has not been tested or modified at all. I need to get this from WCON when I've ported the pipeline. This is because there's no free CWP tier for us to develop on and will have to use WCON for it.

Reply:
Essentially, whatever is not in `bitbucket-pipeline.yml` can be ported as is, which is why it would be nice to have checked here and moved over. But then in WCON I need to recreate bitbucket-pipeline.yml and the CWP specific deploy steps because that's the only place where we can dev and test. Then those files will come back here. Trying to avoid two giant PRs that are very similar if not the same.